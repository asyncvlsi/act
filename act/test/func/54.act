function pfunc (pint a[4]) : pint 
{
  chp {
    self := a[0] + a[1] + a[2] + a[3]
  }
}  

pint w[4];

w[0] = 5;
w[1] = 3;
w[2] = 17;
w[3] = 2;

pint res = pfunc (w);

{ res = 27 : "Should match!" };

{ res = 10 : "What?" };
