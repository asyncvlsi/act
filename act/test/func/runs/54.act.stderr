In expanding ::<Global>
Error on or near line number 19.
*** Assertion failed ***
 assertion: res=10
   message: What?
FATAL: Aborted execution on failed assertion
